libraries/AP_Gripper/AP_Gripper.cpp

```diff
@@ -58,12 +58,12 @@ const AP_Param::GroupInfo AP_Gripper::var_info[] = {
     // @Units: s
     AP_GROUPINFO("REGRAB",  5, AP_Gripper, config.regrab_interval, GRIPPER_REGRAB_DEFAULT),
 
-    // @Param: UAVCAN_ID
+    // @Param: DRONECAN_ID
     // @DisplayName: EPM UAVCAN Hardpoint ID
     // @Description: Refer to https://docs.zubax.com/opengrab_epm_v3#UAVCAN_interface
     // @User: Standard
     // @Range: 0 255
-    AP_GROUPINFO("UAVCAN_ID", 6, AP_Gripper, config.uavcan_hardpoint_id, 0),
+    AP_GROUPINFO("DRONECAN_ID", 6, AP_Gripper, config.uavcan_hardpoint_id, 0),
```

Comment:
I'm not fully sure about the name changing in the parameters since we usually try to avoid breaking old parameter files that were saved (but it obviously won't break the actual tree).

If changing the names though that should happen at the same time we change the `CAN_D1_UC` to not have UC in the name, rather then incrementally bringing it in.

Reply:
I think we should make the parameter CAN_ID instead of DRONECAN_ID.

Reply:
done